protocol.html

```diff
@@ -748,7 +748,15 @@ <h4 property="schema:name">Modifying Resources Using N3 Patches</h4>
+                    <p><span about="" id="server-patch-n3-semantics" rel="spec:requirement" resource="#server-patch-n3-semantics"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> process a patch resource against the target document as follows:</span></span></p>
+                    <ol>
+                      <li>Start from the RDF dataset in the target document, or an empty RDF dataset if the target resources does not exist yet.</li>
+                      <li>If <code>?conditions</code> is non-empty, then the dataset must contain exactly one match of its contents after variable substitution; if not, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.</li>
+                      <li>The resulting (possibly empty) variable mapping is propagated to the <code>?deletions</code> and <code>?insertions</code> formulae.</li>
+                      <li>Then, the resulting triples in <code>?deletions</code> are removed from the RDF dataset if there is exactly one match; if not, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.</li>
```

Comment:
I'm struggling with this one. Is the `409` required for the whole patch if there is not exactly one match for any triple in `?deletions`; or is there to be a `409` for *each* triple that has not exactly one match; or something else? The phrasing I'm suggesting here is probably not correct, but I hope it's closer than the phrasing I'm suggesting be changed...
```suggestion
                      <li>Then, for each resulting triple in <code>?deletions</code> which has exactly one match in the RDF dataset, that match is removed; if there is not exactly one match in the RDF dataset, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.</li>
```

Reply:
Indeed; clarified in 99fcc3219789169e459d7078d42df5e2af9eb64c.